src/views/onboarding/login/index.tsx

```diff
@@ -14,55 +13,78 @@ import {
 import { OnboardingTranslation } from "../context/onboardingTranslation";
 
 import "./styles.scss";
+import { withRouter } from "react-router-dom";
+import { homeRoute } from "../../../shared/routes/routes";
 
-const Login = () => {
+const Login = ({ history }) => {
   const { t: translate } = useTranslation();
 
   // @ts-ignore
-  const { handleChange, values, errors } = useFormik({
+  const { handleChange, values, errors, dirty, isValid } = useFormik({
     initialValues: defaultLoginValues,
     validationSchema: loginValidators,
   });
 
   const submitLogin = async (values) => {
     try {
       console.log(values);
-      await loginRequest(values);
+      const { data } = await loginRequest(values);
+      localStorage.setItem("jwtToken", data["accessToken"]);
```

Comment:
We have localStorageService or something as browserStorageService somewhere in code use that

But anyway we want to use UserStore for this, for now change to browser service that we have, also use const instead of "jwtToken", and name should be accessToken

Reply:
You are using data["accessToken"] since your loginRequest has no type as response, we should add type to it